### DISCUSSION TITLE
Issue starting Airflow in docker (macOS 12.0.1, docker 20.10.11)

### DISCUSSION THREAD

**Discussion by e-wrobel** [apache/airflow#20512] [General]
Hi guys, 

I try to follow the instructions about starting Airflow on my docker environment: https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html. The problem is that airflow web server is throwing error below:
airflow-webserver_1  | 
airflow-webserver_1  |   ____________       _____________
airflow-webserver_1  |  ____    |__( )_________  __/__  /________      __
airflow-webserver_1  | ____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
airflow-webserver_1  | ___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
airflow-webserver_1  |  _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
airflow-webserver_1  | [2021-12-27 11:22:28,953] {dagbag.py:500} INFO - Filling up the DagBag from /dev/null
airflow-webserver_1  | [2021-12-27 11:22:29,229] {manager.py:512} WARNING - Refused to delete permission view, assoc with role exists DAG Runs.can_create User
airflow-webserver_1  | Traceback (most recent call last):
airflow-webserver_1  |   File "/home/airflow/.local/bin/airflow", line 8, in <module>
airflow-webserver_1  |     sys.exit(main())
airflow-webserver_1  |   File "/home/airflow/.local/lib/python3.7/site-packages/airflow/__main__.py", line 48, in main
airflow-webserver_1  |     args.func(args)
airflow-webserver_1  |   File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
airflow-webserver_1  |     return func(*args, **kwargs)
airflow-webserver_1  |   File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
airflow-webserver_1  |     return f(*args, **kwargs)
airflow-webserver_1  |   File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/webserver_command.py", line 368, in webserver
airflow-webserver_1  |     check_if_pidfile_process_is_running(pid_file=pid_file, process_name="webserver")
airflow-webserver_1  |   File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 297, in check_if_pidfile_process_is_running
airflow-webserver_1  |     raise AirflowException(f"The {process_name} is already running under PID {pid}.")
airflow-webserver_1  | airflow.exceptions.AirflowException: The webserver is already running under PID 18.
airflow_airflow-webserver_1 exited with code 1

Do you have any idea what caused that issue or need more data to investigate?

Best regards,
Mr. S

**Comment by Sonins** [comment]
It says webserver is already running. Can you check whether webserver is already running by `docker ps` ?

**Reply by e-wrobel** [reply]
Hi, Yes, I already checked that, I think that issue is related to starting script. I wonder if it's something common?

**Reply by potiuk** [reply]
You just need to stop the other process. Quick start is just that - quick-start, it's not resilient to say starting other instances etc. Just make sure you killed all the processes before starting new. You can do it with `docker down` command or with `docker ps` [manual stuff.](https://www.google.com/search?q=kill+all+docker+containers&oq=kill+all+docker+&aqs=chrome.0.0i512j69i57j0i512l6.4002j0j7&sourceid=chrome&ie=UTF-8) When you play with docker in not fully managed environment, you just need to really, well. manage it yourself.
